crates/storage/store/engines.rs

```diff
@@ -1,5 +1,5 @@
 pub mod api;
-#[cfg(feature = "in_memory")]
+#[cfg(any(feature = "no_libmdbx", feature = "libmdbx"))]
```

Comment:
Is this feature gate needed? It looks like it will always be true

Reply:
Right, not needed.